Source/Core/Core/HW/WII_IPC.h

```diff
@@ -35,6 +35,36 @@ enum StarletInterruptCause
   INT_CAUSE_IPC_STARLET = 0x80000000
 };
 
+enum GPIO : u32
```

Comment:
enum class and provide the necessary overloads.

Reply:
Is the thing I implemented what you wanted?  I'm not *too* experienced with C++, so I don't know if the thing I made in dd5aa9b9ffa1df07e40d68c44e4dba9c4dce71f4 is what you want.